src/openforms/utils/tests/test_decorators.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from django.contrib.messages import get_messages
+from django.contrib.messages.storage.fallback import FallbackStorage
+from django.db import models
+from django.test import TestCase
+from django.test.client import RequestFactory
+from django.utils.translation import gettext_lazy as _
+
+import requests
+import zds_client
+
+from openforms.utils.decorators import dbfields_exception_handler
+
+
+class TestDbFieldsExceptionHandler(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.db_field = models.CharField(
+            _("db field"),
+            blank=True,
+            max_length=1000,
+            help_text=_("help text"),
+        )
+
+    def test_no_exception_detected(self):
+        @dbfields_exception_handler(exceptions=(Exception))
+        def formfield_for_dbfield(db_field, request):
+            return forms.CharField(
+                label="db field",
+                max_length=1000,
+                help_text=_("help text"),
+            )
+
+        request = self.factory.get("/test")
+
+        # setup django.messages
+        setattr(request, "session", "session")
+        messages = FallbackStorage(request)
+        setattr(request, "_messages", messages)
+
+        form_field = formfield_for_dbfield(self.db_field, request)
+
+        messages = [m.message for m in get_messages(request)]
+        self.assertNotIn(
+            "Could not load data for field 'db field' - enable and check the request logs for more details",
+            messages,
+        )
```

Comment:
This is an isolated unittest. We call the function with no side-effects, so nothing else should be writing messages.
I think expecting `get_messages(request) == []` is fine here.